release/platform/platform.go

```diff
@@ -364,10 +364,18 @@ var platforms = []Platform{
 		BuildTags: defaultBuildTags,
 	},
 	{
-		Name:      "macos",
-		Arch:      ArchX86_64,
-		OS:        OSMac,
-		BuildTags: defaultBuildTags,
+		Name:        "macos",
+		VariantName: "macos-arm",
```

Comment:
The docs for VariantName in the Platform struct say setting it should only be done for special builds, and in general we should just use the Name+Arch pair. Unless I'm missing something, I think you should do that here, and remove VariantName.

(I was wondering here if you couldn't have duplicate names, but there are multiple ubuntu1804 names defined here, so I reckon it's ok to just do the normal thing for macos too.)

Reply:
It broke when I had just `macos`, I think because the `Name` has to match Evergreen. I’ll play with it a bit and see if I can get it to work without VariantName.